libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -231,6 +232,11 @@ class NavigationRoute internal constructor(
      */
     val id: String = nativeRoute.routeId
 
+    /**
+     * Describes which router type generated the route.
+     */
+    val origin: RouterOrigin = nativeRoute.routerOrigin.mapToSdkRouteOrigin()
```

Comment:
`property` adds `field` + `getter`, with suggestion adds `getter` only
```suggestion
    val origin: RouterOrigin get() = nativeRoute.routerOrigin.mapToSdkRouteOrigin()
```

Reply:
Why not keep it as a field? This way we don't need to go through the JNI boundary and map the origin type every time we need to access it?